tests/test_bias.py

```diff
@@ -0,0 +1,187 @@
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import hashlib
+import os
+import numpy as np
+import unittest
+
+import lsst.afw.math as afwMath
+import lsst.daf.butler as dafButler
+import lsst.ip.isr as ipIsr
+import lsst.meas.algorithms as measAlg
+import lsst.utils.tests
+
+from lsst.pipe.tasks.repair import RepairTask
+from lsst.utils import getPackageDir
+
+
+class BiasTestCases(lsst.utils.tests.TestCase):
+
+    def setUp(self):
+        """Setup butler, and generate an ISR processed exposure.
+
+        Notes
+        -----
+        DMTN-101 4.1:
+
+        Process an independent bias frame through the ISR including
+        overscan correction and bias subtraction
+
+        """
+        repoDir = os.path.join(getPackageDir('ci_cpp_gen3'), "DATA")
+        butler = dafButler.Butler(repoDir, collections=['raws/latiss', 'calibs/latiss', 'calib/v00'])
+
+        self.config = ipIsr.IsrTaskConfig()
+        self.config.doSaturation = True
+        self.config.doSuspect = True
+        self.config.doSetBadRegions = True
+        self.config.doOverscan = True
+        self.config.doBias = True
+        self.config.doVariance = True
+
+        self.config.doLinearize = False
+        self.config.doCrosstalk = False
+        self.config.doWidenSaturationTrails = False
+        self.config.doBrighterFatter = False
+        self.config.doDefect = False
+        self.config.doSaturationInterpolation = False
+        self.config.doDark = False
+        self.config.doStrayLight = False
+        self.config.doFlat = False
+        self.config.doApplyGains = False
+        self.config.doFringe = False
+        self.config.doMeasureBackground = False
+        self.config.doVignette = False
+        self.config.doAttachTransmissionCurve = False
+        self.config.doUseOpticsTransmission = False
+        self.config.doUseFilterTransmission = False
+        self.config.doUseSensorTransmission = False
+        self.config.doUseAtmosphereTransmission = False
+
+        self.isrTask = ipIsr.IsrTask(config=self.config)
+        # This is not an independent frame.
+        self.raw = butler.get('raw', dataId={'detector': 0, 'exposure': 2020012800007,
+                                             'instrument': 'LATISS'})
+        self.bias = butler.get('bias', dataId={'detector': 0,
+                                               'instrument': 'LATISS',
+                                               'calibration_label': 'bias/ci_cpp_bias'})
+        self.camera = butler.get('camera', dataId={'detector': 0, 'exposure': 2020012800014,
+                                                   'instrument': 'LATISS',
+                                                   'calibration_label': 'unbounded'})
+
+        results = self.isrTask.run(self.raw, camera=self.camera,
```

Comment:
Does this take non-trivial time to run? If it contains all the same answers each time it runs shouldn't it be in `setUpClass` ?